_toc.yml

```
@@ -0,0 +1,58 @@
+format: jb-book
+root: README
+
+parts:
+
+  - caption: About These Tutorials
+    chapters:
+    - file: doc/HowToUse                                                           # TODO
+    - file: environment_set_up/README                                              # TODO: this doc is in-process on another branch
+
+
+  - caption: Common Workflows
```

I think these names should mimic the folder structure
```suggestion
  - caption: Dataset Preprocessing
```